### DISCUSSION TITLE
Colors perfectly accurate. How did you do this?

### DISCUSSION THREAD

**Discussion by uloco** [kovidgoyal/kitty#5787] [General]
I created a color theme for neovim and realized that my beloved iTerm2 does not exactly show the right colors, altough 256 colors / true color support is working. I was using it for such a long time but did not actually realize that all the colors also outside of neovim are slightly different when I check them with the colorpicker.

Now that I know, I know and cannot ignore it haha 🗡️ 

I am also confused in what the correct term is but, kitty has just perfect colors. I can use the same theme and check the colors with the color picker and it is exact or at least so close that you cannot actually see any difference.
Also alacritty does not have this feature although it is also gpu based. So I was really shocked and thought this could not be the ase. Also VimR a gui based neovim application has this exact color values, but iterm and alacritty do not.

I am just curious if this was something that happened because of some specific design choices you made and anything related to this. Also I just wanted to say thank you, this looks really amazing and I am now thinking about switching completely to kitty. Just need a replacement to the global hotkey, top down (quake style) terminal I am used to from iTerm2.

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.macos_colorspace

https://github.com/kovidgoyal/kitty/issues/45
